attributes/default.rb

```diff
@@ -359,6 +359,14 @@
 default['datadog']['process_agent']['container_interval'] = nil
 default['datadog']['process_agent']['rtcontainer_interval'] = nil
 
+# System probe functionality settings
+default['datadog']['system_probe']['enabled'] = nil
+# sysprobe_socket defines the unix socket location
+default['datadog']['system_probe']['sysprobe_socket'] = nil
```

Comment:
I think you can specify the correct default instead of nil

Reply:
Yea I wasn't paying attention to those default values for testing, I added a few.